examples/transforms/rotation.rs

```diff
@@ -0,0 +1,52 @@
+use bevy::prelude::*;
+
+use std::f32::consts::PI;
+
+const FULL_TURN: f32 = 2.0 * PI;
```

Comment:
You can use `TAU` instead of `2.0 * PI`

Reply:
I laugh, but I don't hate the idea :p

Reply:
As most reference materials and tutorials for game and everything uses π, it's probably better to continue using it here instead of τ

Reply:
I'm sticking with PI for now, due to mockersf suggestion.